### PR TITLE
Remove many unused strings

### PR DESCRIPTION
**Pull request by Gymnasiast**
This explicitly does not remove _all_ unused strings, since I believe some have become unused in error while others might be used with string addition. I therefore kept on the safe side.

### CODE REVIEW

File: data/language/en-GB.txt [hunk 1]
```diff
@@ -3152,31 +2396,14 @@ STR_3141    :Multi-circuit per ride not possible with cable lift hill
...
-STR_3149    :{SMALLFONT}{BLACK}Scroll {STRINGID} left/right
-STR_3150    :{SMALLFONT}{BLACK}Scroll {STRINGID} up
-STR_3151    :{SMALLFONT}{BLACK}Scroll {STRINGID} down
-STR_3152    :{SMALLFONT}{BLACK}Scroll {STRINGID} up fast
-STR_3153    :{SMALLFONT}{BLACK}Scroll {STRINGID} down fast
-STR_3154    :{SMALLFONT}{BLACK}Scroll {STRINGID} up/down
-STR_3155    :
-STR_3156    :
-STR_3157    :map
 STR_3158    :graph
```
Comment (AaronVanGeffen) [inline data/language/en-GB.txt]: STR_GRAPH (3158) and STR_LIST (3159) are still assigned to format arguments, but the relevant tooltip is never displayed. I think we should remove those assignments and these two strings, too.

File: data/language/en-GB.txt [hunk 2]
```diff
@@ -3152,31 +2396,14 @@ STR_3141    :Multi-circuit per ride not possible with cable lift hill
...
-STR_3161    :<removed string - do not use>
 STR_3162    :Unable to allocate enough memory
 STR_3163    :Installing new data: 
 STR_3164    :{BLACK}{COMMA16} selected (maximum {COMMA16})
-STR_3165    :
-STR_3166    :{BLACK}(ID: 
 STR_3167    :{WINDOW_COLOUR_2}Includes: {BLACK}{COMMA16} objects
-STR_3168    :{WINDOW_COLOUR_2}Text: {BLACK}{STRINGID}
-STR_3169    :Data for the following object not found: 
+STR_3169    :Data for the following object not found:
```
Comment (AaronVanGeffen) [inline data/language/en-GB.txt]: You've accidentally deleted a trailing space here.

Comment (Gymnasiast) [reply data/language/en-GB.txt]: The IDE ate that space.

File: src/openrct2/localisation/StringIds.h [hunk 3]
```diff
@@ -2481,7 +2481,6 @@ enum
     STR_SHOW_PEOPLE_ON_MAP_TIP = 3143,
     STR_SHOW_RIDES_STALLS_ON_MAP_TIP = 3144,
 
-    STR_GRAPH = 3158,
     STR_LIST = 3159,
```
Comment (AaronVanGeffen) [inline src/openrct2/localisation/StringIds.h]: You've removed this string, but not the string id :-)

**Pull request merged**